webroot/js/cake-core.js

```diff
@@ -0,0 +1,101 @@
+var postLinkFunction = function () {
```

Comment:
This file is introducing global variables. Since it's self contained you could just wrap the whole file in an IIFE `(function () { ... }());`